ProcessLib/HeatTransportBHE/BHE/BHE_1U.h

```diff
@@ -46,6 +46,7 @@ class BHE_1U final : public BHECommon
            PipeConfiguration1U const& pipes);
 
     static constexpr int number_of_unknowns = 4;
+    static constexpr int number_of_grout_zones = 2;
```

Comment:
This PR implements coaxial BHE processes with types of CXA and CXC. Why is it necessary to add this line to the type BHE_1U?

Reply:
In the BHE model, the total heat flux through the grout is separated into two parts in 1U type because there is grout around inlet pipe and outlet pipe, respectively. Thus, the total heat flux through the grout should be the sum of these two parts. However, for coaxial BHE, there only one grout around the outer pipe. It is necessary to add this line because the heat flux through grout for different types was not considered before. The number of grout zone was just considered in the local assembler before, which wound cause double counting the total heat flux through grout for coaxial BHE.